### DISCUSSION TITLE
autoscaler: cx11 removal issues

### DISCUSSION THREAD

**Discussion by pat-s** [mysticaltech/terraform-hcloud-kube-hetzner#1514] [Q&A] [answered]
Not sure if a local issue or a general one: but starting today, I had to override the `draining-node-pool` instance type from the deprecated `cx11` to `cx22` to make the autoscaler happy again (otherwise all scale up/down operations weren't triggered), *even though* I was using the official autoscaler image 1.30.2.

```tf
    {
        name        = "draining-node-pool"
        server_type = "cx22"
        location    = "nbg1"
        min_nodes   = 0
        max_nodes   = 0
    },
```

cc @apricote as I think this should actually work OOB with 1.30.2?

**Comment by apricote** [comment, marked as answer]
I made a mistake in the first version list, please upgrade to 1.30.3 instead of 1.30.2.

**Reply by pat-s** [reply]
Ah! Didn't know 1.30.3 even existed as there was no release for it. [This link](https://explore.ggcr.dev/?repo=registry.k8s.io%2Fautoscaling%2Fcluster-autoscaler) helps to list all versions.
